config/kubernetes/sig-azure/teams.yaml

```diff
@@ -1,5 +1,5 @@
 teams:
-  sig-azure:
+  provider-azure:
```

Comment:
This will need the `previously` field everywhere: https://github.com/kubernetes/community/blob/master/github-management/org-owners-guide.md#structure

(sorry for the extra work!)

Reply:
Thanks for catching that. Fixed!